packages/components/src/core/DropdownMenu/index.tsx

```diff
@@ -86,14 +99,36 @@ const DropdownMenu = <
     options,
     onClickAway = noop,
     ClickAwayListenerProps,
-    width = 160,
+    width = MINIMUM_DROPDOWN_MENU_POPPER_WIDTH,
     ...rest
   } = props;
 
   const isMultiColumn = options && !!options[0] && "options" in options[0];
-  const singleColumnPadding = `${theme.app?.spacing?.xs}px 0 ${theme.app?.spacing?.xs}px ${theme.app?.spacing?.xs}px !important`;
-  const multiColumnNoSearchOrTitlePadding = `${theme.app?.spacing?.l}px ${theme.app?.spacing?.xxs}px ${theme.app?.spacing?.l}px ${theme.app?.spacing?.l}px !important`;
-  const multiColumnWithSearchOrTitlePadding = `${theme.app?.spacing?.xs}px 0 ${theme.app?.spacing?.l}px ${theme.app?.spacing?.xs}px !important`;
+
+  // (masoudmanson): The DropdownMenu's Popper component should have a minimum
+  // width of MINIMUM_DROPDOWN_MENU_POPPER_WIDTH pixels if the DropdownMenu is
+  // a single column dropdown.However, for larger multi-column autocompletes,
+  // the width should be set to 'auto' to accommodate the expanded layout.
```

Comment:
ahh thanks for the comment! All makes sense now!

Reply:
🥳